<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Recent Projects – Juan F Poyatos</title>
    <style>
        @import url('https://fonts.googleapis.com/css2?family=Baskerville&display=swap');

        body {
            font-family: "Baskerville", serif;
            text-align: justify;
            background-color: #A9A9A9;
            color: black;
            padding: 50px;
        }

        .container {
            max-width: 600px;
            margin: auto;
            background: #D3D3D3;
            padding: 20px;
            border-radius: 10px;
        }

        h1 {
            font-size: 1.8em;
            margin-bottom: 30px;
        }

        .project {
            margin-bottom: 30px;
        }

        .project-title {
            font-weight: bold;
            font-size: 1.1em;
            margin-bottom: 1px;
        }

        p {
            font-size: 1em;
        }

        a {
            color: black;
            text-decoration: none;
            font-weight: bold;
        }

        a:hover {
            text-decoration: underline;
        }
    </style>
</head>

<body>
    <div class="container">
        <h1>Recent Projects</h1>

        <div class="project">
            <div class="project-title">Capturing developmental context with large language models</div>
            <p>Development is a <i>contextual</i> process where the state of a cell is changing along the organismal maturation. 
                We can now follow this process at the single cell level and quantify the activity of genes on each cell. We want then to 
                understand how the context of a cell influences the activity of genes, and how this can be captured by large language models. 
                Some first efforts with Zebrafish data <a href="https://www.biorxiv.org/content/10.1101/2025.07.09.663853v1" style="color: steelblue;font-weight: normal;">here</a>.</p>   
            </p>
        </div>

        <div class="project">
            <div class="project-title">A developmental-phenotypic rule</div>
            <p> It is commonly argued that genes that are active in similar ways during development tend to produce similar effects on an organism’s traits. 
                But is this really the case? To test this idea, we are analyzing gene activity in different cell types during the development of the nematode 
                <i>Caenorhabditis elegans</i>, using a detailed single-cell gene expression atlas. We then compare these patterns to a curated database of phenotypes, observable 
                traits or behaviors linked to specific genes.
                This is what we <a href="https://www.biorxiv.org/content/10.1101/2025.01.20.633901v1" style="color: steelblue;font-weight: normal;">found</a>.</p>

            </p>
        </div>

        <div class="project">
            <div class="project-title">Polygenic prediction and gene regulation networks</div>
            <p>We aim to know how well genetic data can predict complex traits, like behavior or disease risk. Traditional models assume gene effects add up simply, 
                but biology is often nonlinear and network-driven. Using simulations of gene regulatory networks, we explore when these assumptions hold, and when they break down. 
                Our work helps clarify the limits of genetic prediction tools like polygenic scores. 
                <a href="https://royalsocietypublishing.org/doi/10.1098/rsos.241992" style="color: steelblue;font-weight: normal;">Here</a>, you can find details. </p>
        </div>

        <div class="project">
            <div class="project-title">Protein metabolic costs in human cells and tissues</div>
            <p>We explore the idea that cells and tissues manage protein production to minimize energy costs. By analyzing highly expressed proteins across human cells, 
                tissues, and tumors, we show that this metabolic efficiency principle helps explain which proteins are favored, and when this pattern breaks down. 
                This approach offers a tractable way to understand the energetic logic behind both normal cellular function and disease. Want to read more? 
                <a href="https://academic.oup.com/pnasnexus/article/4/1/pgaf008/7958171" style="color: steelblue;font-weight: normal;">Here</a>. </p>
        </div>

        <div class="project">
            <div class="project-title">Design principles of multi-map variation</div>
            <p>Biology involves layered mappings -from genes to traits to fitness- all influenced by variation. This variation can be masked, redirected, or amplified, shaping robustness, plasticity, and evolution. 
                I propose <a href="https://iopscience.iop.org/article/10.1088/1478-3975/ad5d6c" style="color: steelblue;font-weight: normal;">here</a> that general <i>design principles </i> govern 
                how variation moves through these layers, a concept I call multi-map variation. Identifying these principles may help us better understand and predict evolutionary change.</p>
        </div>

        <p><a href="index.html">← Back to Home</a></p>
    </div>
</body>
</html>
